relacy/relacy_std.hpp

```diff
@@ -19,64 +19,68 @@
 
 namespace std
 {
-    using rl::memory_order;
-    using rl::mo_relaxed;
-    using rl::mo_consume;
-    using rl::mo_acquire;
-    using rl::mo_release;
-    using rl::mo_acq_rel;
-    using rl::mo_seq_cst;
+    namespace rlimp {
+      using namespace rl;
+    }
 
-    using rl::atomic;
-    using rl::atomic_thread_fence;
-    using rl::atomic_signal_fence;
+    #define memory_order rlimp::memory_order

```

Comment:
# must be the first character on a line according to C++ standard
if you want to indent, use:
```
#   define ...
```